lib/src/Uncertainty/Distribution/MarshallOlkinCopula.cxx

```diff
@@ -142,7 +142,6 @@ Bool MarshallOlkinCopula::hasIndependentCopula() const
 void MarshallOlkinCopula::setAlpha(const Scalar alpha)
 {
   if ((alpha < 0.0) || (alpha > 1.0)) throw InvalidArgumentException(HERE) << "Alpha MUST be in [0, 1], here alpha=" << alpha;
```

Comment:
if (!(alpha >= 0.0) && !(alpha <= 1.0))

We should use the "common" way for checking arguments

Reply:
Ok. This way is indeed used in:
https://github.com/openturns/openturns/blob/cdc0df551b77e9f4dcfafc85b2d4206a90e0fad3/lib/src/Uncertainty/Distribution/WeibullMax.cxx#L344
but not in:
https://github.com/openturns/openturns/blob/cdc0df551b77e9f4dcfafc85b2d4206a90e0fad3/lib/src/Uncertainty/Distribution/AliMikhailHaqCopula.cxx#L374

I will fix this the way you suggest.